palm/pilrc/Portfile

```diff
@@ -16,21 +16,15 @@ long_description	This is a tool that turns a textual description of \
 					prc-tools and CodeWarrior for Palm OS.
 homepage		http://pilrc.sourceforge.net/
 master_sites	sourceforge
-checksums		md5 bbfea963c4e34252f634c6a39a2578b8
+checksums		rmd160  e9dfb96082c7406e269ea99aec4ce2a6029e8afd \
```

Comment:
Total nitpick, can you align the checksums?

Reply:
On my screen they are aligned. Looks like this file uses ts=4. Is there a standard I should retab it to?

Reply:
Answered my own question, the standard appears to be 4 spaces.